### PR TITLE
Update InfoResponse

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/tests/client/client_builder.rs [hunk 1]
```diff
@@ -52,7 +52,9 @@ async fn client_builder() {
                 "vByteFactorData":1,
                 "vByteOffset":380
```
Comment (Thoralf-M) [inline sdk/tests/client/client_builder.rs:53]: This doesn't exist anymore?

Comment (thibault-martinez) [reply sdk/tests/client/client_builder.rs:53]: Right, I'll remove it in develop and will rebase 2.0 at some point.

Comment (thibault-martinez) [reply sdk/tests/client/client_builder.rs:53]: https://github.com/iotaledger/iota-sdk/pull/649

File: sdk/src/client/node_manager/syncing.rs [hunk 2]
```diff
@@ -104,7 +104,8 @@ impl ClientInner {
             if let Some((info, _node_url)) = nodes.first() {
                 let mut network_info = self.network_info.write().await;
 
-                network_info.latest_milestone_timestamp = info.status.latest_milestone.timestamp;
+                // TODO change to one of the new timestamps, which ones ?
+                // network_info.latest_milestone_timestamp = info.status.latest_milestone.timestamp;
```
Comment (Alex6323) [inline sdk/src/client/node_manager/syncing.rs:108]: just marking this so it doesn't go unnoticed (is this solved with this PR?)

Comment (thibault-martinez) [reply sdk/src/client/node_manager/syncing.rs:108]: I've opened an issue already https://github.com/iotaledger/iota-sdk/issues/647

**Review by Alex6323 — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

File: sdk/src/client/node_manager/syncing.rs (hunk 2 above)
Comment (thibault-martinez) [reply sdk/src/client/node_manager/syncing.rs:108]: @Alex6323 Please resolve :trollface:

**Review by qrayven — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**